internal/mockstore/mocktikv/errors.go

```diff
@@ -155,3 +155,14 @@ func (e *ErrAssertionFailed) Error() string {
 	return fmt.Sprintf("AssertionFailed { StartTS: %v, Key: %v, Assertion: %v, ExistingStartTS: %v, ExistingCommitTS: %v }",
 		e.StartTS, hex.EncodeToString(e.Key), e.Assertion.String(), e.ExistingStartTS, e.ExistingCommitTS)
 }
+
+// ErrAssertionFailed is returned if any assertion fails on a transaction request.
+type LockIfExistsFailed struct {
```

Comment:
Please fix the lint, and consider prefixing the type name with `Err` to keep naming consistent with other error types here.

Reply:
Actually since there's not any specific protobuf type for this kind of error, and it's a mock implementation, Personally I think it's not very necessary to define a special error type for this case. Just returnning errors.New(...) is enough.